plugins/reach/lib/api/data/entryVendorTaks/KalturaIntelligentTaggingVendorTaskData.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * @package plugins.reach
+ * @subpackage api.objects
+ * @relatedService EntryVendorTaskService
+ */
+class KalturaIntelligentTaggingVendorTaskData extends KalturaVendorTaskData
+{
+
+    /**
+     * Optional - The id of the caption asset object
+     * @insertonly
+     * @var string
+     */
+    public $assetId;
+
+    private static $map_between_objects = array
+    (
+        'assetId',
+    );
+
+    /* (non-PHPdoc)
+     * @see KalturaCuePoint::getMapBetweenObjects()
+     */
+    public function getMapBetweenObjects()
+    {
+        return array_merge(parent::getMapBetweenObjects(), self::$map_between_objects);
+    }
+
+    /* (non-PHPdoc)
+    * @see KalturaObject::toObject($object_to_fill, $props_to_skip)
+    */
+	public function toObject($dbObject = null, $propsToSkip = array())
```

Comment:
No need to override both toInsertableObject and toObject (one calls the other) , use only toInsertableObject